templates/candidate/_sidebar.html.twig

```diff
@@ -0,0 +1,18 @@
+{% if isCandidate or is_granted('HAS_DELEGATED_ACCESS_ANY') %}
```

Comment:
why we check isCandidate here? this template is used on `candidate/layout`, so we know that the user is already candidate, no?

Reply:
Yes, it has been removed.